index.html

```diff
@@ -290,7 +290,23 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
-
+      <footer class="footer">
+        <div class="footer-div">
+          <div class="fheading">
+            <h2 class="contact-me text-white text-center">Contact me</h2>
+            <p class="para text-white">If you have an application you are interested in developing, a feature that you need build or a project that needs coding. I'd love to help with it.</p>
+          </div>
+          <form action="https://formspree.io/f/xrgranwn" class="form" method="POST">
+            <input name="username" class="mb-4 uname" id="name" type="text" placeholder="your name">
```

Comment:
Looks good, but I recommend you adding attributes to validate the data in your input for username.

- [ ] Add validation into `input` for example `maxlength="30"`.